lock-farm/deploy/Deploy-LockFarm.ts

```diff
@@ -83,14 +77,22 @@ const deploy: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 
   await waitSeconds(60);
   // Deploy Emissionor
-  // const treasuryAddress = '0x724c670d0215cC75d990b7D28604A84E1F1caa1c';
-  // const rewardTokenAddress = '0xBa5B18A16d54a9687EFE5eeEbc15c223b575aBfd';
   const emissionor = await deployEmissionor(
     treasury.address,
     splitter.address,
     rewardTokenAddress
   );
   console.log('Emissionor: ', emissionor.address);
+  try {
+    await emissionor.initialize(
+      emitStartTimestamp,
```

Comment:
@bejaxer  will you update your Start Date to be 11th October CET @1pm for the New Emission?

Reply:
`export const emitStartTimestamp = 1665486000; /// Tue Oct 11 2022 11:00:00 GMT+0000`